src/util.ml

```diff
@@ -41,8 +41,14 @@ value search_in_lang_path = search_in_path Secure.lang_path;
 
 value start_with_vowel s =
   if String.length s > 0 then
-    match Char.lowercase s.[0] with
-    [ 'a' | 'e' | 'i' | 'o' | 'u' -> True
+    let c = 
+      match s.[0] with
+        [ '<' -> let k = String.index_from s 0 '>' in
```

Comment:
There are cases where some html tag is between the "de" and the word starting with a vowel.
We need to hunt for the vowel jumping above such a tag!